### ISSUE TITLE
x-axis text is snipped at the right and does not show the minutes

### ISSUE THREAD

**Issue by parakite** [0xB10C/memo#73]
![gui block timeline](https://user-images.githubusercontent.com/70532411/95683099-c470dc80-0c06-11eb-89d9-a3148ae372a3.PNG)

The x-axis time display is bit of wrong at the right-most, where it only prints the hour ( 21:) and not the minutes.

This can be improved.

**Comment by 0xB10C** [comment]
Good point, this can be improved. I think it's an issue with the library I used though. Have been thinking about redoing some of these charts with a different, more flexible library. Will keep this issue open for now.
